Docs: explain how to override default user avatar image (#14127)
wagtail/wagtail: docs/advanced_topics/customization/admin_templates.md

Additionally, you can use the default `size` parameter that is passed in to the hook if you need to attach it to a request or do any further processing on your image.

### Overriding the default user avatar image

> Override `wagtailadmin/images/default-user-avatar.png` by placing your custom file at the same path within one of your apps (see Django’s [`INSTALLED_APPS`](inv:django:std:setting#INSTALLED_APPS) for ordering).

Reviewer: Nipick:

```suggestion
To change the default user avatar image, override `wagtailadmin/images/default-user-avatar.png` by placing your custom avatar image at the same path within one of your apps (see Django’s [`INSTALLED_APPS`](inv:django:std:setting#INSTALLED_APPS) for ordering).
```

Reviewer: Side note: for this to work, the project should not have modified [`STATICFILES_FINDERS`](https://docs.djangoproject.com/en/6.0/ref/settings/#std-setting-STATICFILES_FINDERS), or if it did, `AppDirectoriesFinder` should be in the list.

Author: Thankyou for the  review ! I have commited your suggestion and added the note for `AppDirectoriesFinder` in `STATICFILES_FINDERS`.

Reviewer 2: I don't think it warrants a specific mention. People could even use the project-wide directories in `STATICFILES_DIRS`, so it doesn't have to be an app. If anything, mentioning `STATICFILES_FINDERS`/`AppDirectoriesFinder` could sidetrack the reader's attention for something that they likely haven't even changed.

I think a better way to frame this is by saying something along the lines of, the file should be put in one of the static directories with a higher precedence than the `wagtail.admin` app. Maybe with a link to https://docs.djangoproject.com/en/6.0/howto/static-files/ which should be a good starting point to understand how static files are found.

Author: removed the finder-specific note and rephrased it to focus on static file precedence, with a link to django’s static files docs

> To change the default user avatar image, override `wagtailadmin/images/default-user-avatar.png` by placing your custom avatar image in a  [`static file directory`](inv:django#howto/static-files/index) with higher precedence than `wagtail.admin` (see Django's [`INSTALLED_APPS`](inv:django:std:setting#INSTALLED_APPS) for ordering).

Reviewer 2: ```suggestion
To change the default user avatar image, override `wagtailadmin/images/default-user-avatar.png` by placing your custom avatar image in a [`static file directory`](inv:django#howto/static-files/index) with higher precedence than `wagtail.admin` (see [`INSTALLED_APPS`](inv:django:std:setting#INSTALLED_APPS) in your Django settings for ordering).
```

> 
> You can verify the override with [`findstatic`](inv:django:std:django-admin#findstatic):
> ```
> python manage.py findstatic wagtailadmin/images/default-user-avatar.png
> ```

Reviewer 2: Add a newline before the code block

````suggestion
You can verify the override with [`findstatic`](inv:django:std:django-admin#findstatic):

```
python manage.py findstatic wagtailadmin/images/default-user-avatar.png
```
````

Additionally, you can use the default `size` parameter that is passed in to the hook if you need to attach it to a request or do any further processing on your image.

### Overriding the default user avatar image

> To change the default user avatar image, override `wagtailadmin/images/default-user-avatar.png` by placing your custom avatar image in a [`static file directory`](inv:django#howto/static-files/index) with higher precedence than `wagtail.admin` (see [`INSTALLED_APPS`](inv:django:std:setting#INSTALLED_APPS) in your Django settings for ordering).

Reviewer 2: ```suggestion
To change the default user avatar image, override `wagtailadmin/images/default-user-avatar.png` by placing your custom avatar image in a [`static` file directory](inv:django#howto/static-files/index) with higher precedence than `wagtail.admin` (see [`INSTALLED_APPS`](inv:django:std:setting#INSTALLED_APPS) in your Django settings for ordering).
```

Additionally, you can use the default `size` parameter that is passed in to the hook if you need to attach it to a request or do any further processing on your image.

> (custom_user_interface_fonts)=

Reviewer 2: Please don't remove this.

Author: That was removed by mistake sorry for that restoring in a moment

Author: updated
